arch/x86/core/early_serial.c

```diff
@@ -99,4 +107,10 @@ void z_x86_early_serial_init(void)
 
 	/* Turn on FIFO. Some hardware needs this before transmitting */
 	OUT(REG_FCR, FCR_FIFO | FCR_FIFO_1 | FCR_RCVRCLR | FCR_XMITCLR);
+
+	early_serial_init_done = true;
+	if (suppressed_chars) {
```

Comment:
I need one empty line before the if, since the assigned variable is not the one being tested.

Reply:
fixed